angular/src/app/book-table/book-table.component.html

```diff
@@ -81,60 +78,53 @@ <h3>{{'bookTable.title' | translate}}</h3>
-          
-          <div  flex-gt-xs="50">
+
+          <div flex-gt-xs="50">
             <form class="pad" [formGroup]="invitationForm">
               <div layout="column" class="push-bottom" flex>
                 <span class="title-card property-text-bold text-upper">{{'bookTable.reservationTitle' | translate}}</span>
               </div>
 
-              <div layout= "column" flex>
-                <md2-datepicker 
-                  formControlName="bookingDate" 
-                  openOnFocus="true" 
-                  type="datetime" 
-                  container="dialog" 
-                  [min]="minDate" 
-                  [placeholder]="'formFields.dateTime' | translate">
-                </md2-datepicker>
-                
+              <div layout="column" flex>
+                <mat-form-field color="accent" flex>
+                  <input matInput [placeholder]="'formFields.dateTime' | translate" formControlName="bookingDate" [min]="minDate" [owlDateTimeTrigger]="booking2"
+                    [owlDateTime]="booking2">
+                </mat-form-field>
+                <owl-date-time [pickerMode]="'dialog'" #booking2></owl-date-time>
+
                 <mat-form-field color="accent" flex>
                   <input matInput [placeholder]="'formFields.name' | translate" formControlName="name">
                   <mat-error *ngIf="invName.invalid && (invName.dirty || invName.touched)">
                     <div *ngIf="invName.errors.required">
-                       {{'bookTable.formErrors.nameRequired' | translate}}
+                      {{'bookTable.formErrors.nameRequired' | translate}}
                     </div>
                   </mat-error>
                 </mat-form-field>
-                
+
                 <mat-form-field color="accent" flex>
                   <input matInput [placeholder]="'formFields.email' |  translate" formControlName="email">
                   <mat-error *ngIf="invEmail.invalid && (invEmail.dirty || invEmail.touched)">
                     <div *ngIf="invEmail.errors.required">
                       {{'bookTable.formErrors.emailRequired' | translate}}
                     </div>
                     <div *ngIf="invEmail.errors.pattern">
-                       {{'bookTable.formErrors.emailPattern' | translate}}
+                      {{'bookTable.formErrors.emailPattern' | translate}}
                     </div>
                   </mat-error>
                 </mat-form-field>
               </div>
 
               <h4>{{'formFields.guests' | translate}}</h4>
-              <td-chips
-                class="pad-bottom-sm"
-                formControlName="invitedGuests"
-                name="invitedGuests"
-                (add)="validateEmail()"
-                [placeholder]="'formFields.guestsPH' | translate">
+              <td-chips class="pad-bottom-sm" formControlName="invitedGuests" name="invitedGuests" (add)="validateEmail()" [placeholder]="'formFields.guestsPH' | translate">
```

Comment:
To be honest I would try to keep formatting consistent (i preferred the one before - if there are many attributes, then it is much more clear to list those line by line, later also easier for git to find differences when one attribute is added / changed)

Reply:
Ok, Agree. Well, to be honest this line has been modified by prettier, but I am committing it with your suggestion.